### PR TITLE
Fix Typos and Broken Link in Docs

### PR DESCRIPTION
**Pull request by acham1**
**Check List**
- [x] Tests has been run in packages where changes made if available
- [x] Linter has been run for changed code
- [x] Tests for the changes have been added if not covered yet
- [x] Docs have been added / updated if required

**Issue Reference this PR resolves**

N/A

**Description of Changes Made (if issue reference is not provided)**

Just some minor typo fixes, thanks!

### CODE REVIEW

File: docs/content/Caching/Using-Pre-Aggregations.md [hunk 1]
```diff
@@ -97,6 +97,7 @@ variables.
 
 [ref-config-extdbtype]: /config#options-reference-external-db-type
 [ref-config-extdriverfactory]: /config#options-reference-external-driver-factory
+[link-production-checklist-refresh]: /deployment/production-checklist#set-up-refresh-worker
```
Comment (acham1) [inline docs/content/Caching/Using-Pre-Aggregations.md:100]: Fixes the link on line 67

Comment (hassankhan) [reply docs/content/Caching/Using-Pre-Aggregations.md:100]: Great catch @acham1

**Review by hassankhan — APPROVED** [review]
Awesome work @acham1 🙌 thank for taking time to make a contribution 😄

**Comment by acham1** [comment]
Thanks Hassan; awesome presentation at Community Call yesterday!

I'm looking forward to more docs on running cube store in production!

**Comment by rpaik** [comment]
Thanks @acham1 😄

**Pull request merged**